client/components/itinerary-form.jsx

```diff
@@ -0,0 +1,148 @@
+        this.setState({
+          placeId,
+          address,
+          name,
+          rating,
+          website,
+          phoneNum,
+          hours,
+          numOfRatings,
+          adrAddress
+        });
+      })
+      .catch(err => console.error(err));
+  }
+
+  handleChange(event) {
+    const { name, value } = event.target;
+    this.setState({ [name]: value });
+  }
+
+  handleAutocompleteChange(event) {
+    this.setState({ address: event });
+  }
+
+  handleSubmit(event) {
+    event.preventDefault();
+
+    const form = event.target;
+    const req = {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(this.state)
+    };
+    fetch('api/itinerary', req)
+      .then(res => res.json())
+      .then(result => {
+        window.location.hash = `#trip-itinerary?tripId=${this.state.tripId}`;
+      });
+
+    form.reset();
+  }
+
+  render() {
+    return (
+      <>
+        <form onSubmit={this.handleSubmit}>
+          <div className="row flex-column">
+            <label htmlFor="destination">
+              Search Places
+            </label>
+            <Autocomplete onChange={this.handleAutocompleteChange} onSelect={this.handleSelect} name="address" address={this.state.address} />
+          </div>
+          <div className="row flex-column day-margin">
+            <label htmlFor="destination">
+              Date
+            </label>
+            <input
+              required
+              id="date"
+              type="date"
+              name="date"
+              onChange={this.handleChange}
+              className="itinerary-input poppins">
+            </input>
+          </div>
+          <div className="row flex-column day-margin">
+            <label htmlFor="destination">
+              Start Time
+            </label>
+            <input
+              required
+              id="startTime"
+              type="time"
+              name="startTime"
+              onChange={this.handleChange}
+              className="itinerary-input poppins">
+            </input>
+          </div>
+          <div className="row flex-column day-margin">
+            <label htmlFor="destination">
+              End Time
+            </label>
+            <input
+              required
+              id="endTime"
+              type="time"
+              name="endTime"
+              onChange={this.handleChange}
+              className="itinerary-input poppins">
+            </input>
+          </div>
+          <div className="row justify-space-between header-margin">
+            <a className="cancel-button flex justify-center align-center" href={`#trip-itinerary?tripId=${this.props.tripId}`}>Cancel</a>
+            <button className="add-save-button poppins">Add</button>
+          </div>
+        </form>
+      </>
+    );
+  }
+}
+
+export default GoogleApiWrapper({
+  apiKey: ('AIzaSyADlDyACi5WP6xynptx0Au3ZXC1xhzBTRg')
+})(ItineraryForm);
```

Comment:
As we discussed, you should access this value at `process.env.GOOGLE_MAPS_API_KEY`